Makefile

```diff
@@ -27,7 +27,7 @@ BINDIR        ?= bin
 BUILD_DIR     ?= build
 COVERAGE      ?= $(CURDIR)/coverage.html
 SC_PKG         = github.com/kubernetes-incubator/service-catalog
-TOP_SRC_DIRS   = cmd contrib pkg
+TOP_SRC_DIRS   = cmd contrib pkg plugin
```

Comment:
it calls-out the idea that admission controllers are not all required, and if need be, they may not all end up being compiled into tree for each type of build target.  it also aligns with what i think should be a goal of the project where code is consistently structured with the upstream kube where possible to ease developer entry across core kube and catalog.